src/Compilers/CSharp/Portable/Symbols/ExtendedErrorTypeSymbol.cs

```diff
@@ -89,15 +91,15 @@ internal ExtendedErrorTypeSymbol AsUnreported()
         private static ImmutableArray<Symbol> UnwrapErrorCandidates(ImmutableArray<Symbol> candidateSymbols)
         {
             var candidate = candidateSymbols.IsEmpty ? null : candidateSymbols[0] as ErrorTypeSymbol;
-            return ((object)candidate != null && !candidate.CandidateSymbols.IsEmpty) ? candidate.CandidateSymbols : candidateSymbols;
+            return ((object?)candidate != null && !candidate.CandidateSymbols.IsEmpty) ? candidate.CandidateSymbols : candidateSymbols;
```

Comment:
>candidate [](start = 29, length = 9)

nit: `is` check instead?

Reply:
I intentionally avoided extraneous code style changes.